server/app.js

```diff
@@ -6,44 +6,52 @@ import webpackMiddleware from 'webpack-dev-middleware';
 import webpackHotMiddleware from 'webpack-hot-middleware';
 import webpackConfig from '../webpack.config';
 
-
 import userRoute from './route/apiRoute';
```

Comment:
Also, I think it'd have been best to have a dedicated server for your express app that does not include webpack. You could then create another server for the client side (if need be). That way, your server and client are decoupled and you are able to use the server for other frontend apps if you so desire.